libs/configuration-variables-resolver/tests/VariablesResolver/ConfigurationVariablesResolverTest.php

```diff
@@ -0,0 +1,582 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ConfigurationVariablesResolver\Tests\VariablesResolver;
+
+use Keboola\ConfigurationVariablesResolver\ComponentsClientHelper;
+use Keboola\ConfigurationVariablesResolver\Exception\UserException;
+use Keboola\ConfigurationVariablesResolver\VariablesRenderer\MustacheRenderer;
+use Keboola\ConfigurationVariablesResolver\VariablesResolver\ConfigurationVariablesResolver;
+use Keboola\StorageApi\Client as StorageClient;
+use Keboola\StorageApi\Components;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\Options\Components\Configuration as StorageConfiguration;
+use Keboola\StorageApi\Options\Components\ConfigurationRow;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\StorageApiBranch\Factory\ClientOptions;
+use Monolog\Handler\TestHandler;
+use Monolog\Logger;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class ConfigurationVariablesResolverTest extends TestCase
```

Comment:
Cely tenhle test je upraveny puvodni test variables resolveru (https://github.com/keboola/platform-libraries/blob/main/libs/configuration-variables-resolver/tests/VariableResolverTest.php), coz beru jako jakous takous zaruku, ze to v zakladu melo fungovat stejne, jako do ted.